### DISCUSSION TITLE
Integrate two accounts on qfield cloud

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3206] [Q&A]
I would like to know is there a provision of linking two accounts on app.qfield.cloud. 
For example, If I am having a free account with a email id and second pro account with different emai Id . Will I be able to link to accounts in terms of sharing projects, number of collaborators etc.?

**Comment by suricactus** [comment]
If you have public projects, there is no limitation.

If you have private project, you can have up to one PRO user as a collaborator per project.

Yhe best way to do it to use organization, which can have projects with as many collaborators (PRO or non-PRO). The billing is per active user per month and for your used project storage.

**Reply by suricactus** [reply]
Related question https://github.com/opengisch/QField/discussions/3204
